star-rx72n-firmware/lib/rx_hcsr04/src/rx_hcsr04_hal_hw.c

```
@@ -39,22 +39,20 @@
  * @return k_rx_ok on success
  * @return k_rx_err_invalid_arg if pin is out of range
  */
-static rx_err_t internal_validate_port_pin(const rx_port_pin_t pin,
-                                           uint8_t*            port,
-                                           uint8_t*            pin_num)
+static rx_err_t internal_validate_port_pin(const rx_port_pin_t pin, uint8_t* port, uint8_t* pin_num)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'internal_validate_port_pin' of similar type ('int *') are easily swapped by mistake

(bugprone-easily-swappable-parameters)